shared/css/user-menu.scss

```diff
@@ -72,6 +72,61 @@
     a + a {
       border-top: 1px solid $charcoal;
     }
+
+    &.hide-responsive-menu {
+      display: none;
+    }
+  }
+}
+
+.help_button {
+  padding-left: 15px;
+  padding-top: 4px;
+
+  &.show-mobile {
+    display: none;
+  }
+  .show-mobile {
+    display: none;
+  }
+  &.show-always {
+    display: block;
+  }
+  .show-mobile {
+    display: none;
+  }
+  display: inline-block;
+  float: left;
+  z-index: 99;
+  position: relative;
```

Comment:
tiny nit: should these lines be indented?

Reply:
I don't think so but I'll add an extra new line to separate them from all the unrelated stuff right above.